docs/calico-with-docker/docker-network-plugin/ManualSetup.md

```diff
@@ -37,13 +37,13 @@ up a cluster.
 > NOTE: If you are running etcd with SSL/TLS, see the (Etcd Secure Cluster)[../../EtcdSecureCluster.md]
 > documentation.
 
-### Docker
+### Docker (with multi-host networking)
 
 Follow the instructions for installing [Docker][docker].  A version of 1.9 or
 greater is required.
 
-To use the multi-host native networking feature of Docker, the Docker daemon
-needs to be run specifying a cluster store.  If using etcd as a cluster store,
+To use Calico as a Docker network plugin, the Docker daemon
```

Comment:
Yes I think we do need to talk about using daemon.json somewhere - quite often people aren't aware of it because in general they don't need to provide additional docker parms.  This seems like the right place for this information.  If you wanted to keep it simple then it would be better to remove the docker run parm - since I think daemon.json is the preferred way to go now... only problem there is that we'd probably want to update anywhere we mention the --cluster-store parameter.